datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/tag/TagType.java

```diff
@@ -59,6 +89,64 @@ public EntityType type() {
         }
     }
 
+    @Override
+    public SearchResults search(@Nonnull String query,
+                                @Nullable List<FacetFilterInput> filters,
+                                int start,
+                                int count,
+                                @Nonnull QueryContext context) throws Exception {
+        final Map<String, String> facetFilters = ResolverUtils.buildFacetFilters(filters, TAG_SEARCH_CONFIG.getFacetFields());
+        final CollectionResponse<com.linkedin.tag.Tag> searchResult = _tagClient.search(query, null, facetFilters, null, start, count);
+        return SearchResultsMapper.map(searchResult, TagMapper::map);
+    }
+
+    @Override
+    public Tag update(@Nonnull TagUpdate input, @Nonnull QueryContext context) throws Exception {
+        // TODO: Verify that updater is owner.
+        final CorpuserUrn actor = new CorpuserUrn(context.getActor());
```

Comment:
I think getActor() may return an URN string if I remember correctly... the "actor" cookie value, which is an urn string

Reply:
Which means we may want to use

```actor = CorpuserUrn.createFromString(context.getActor())```

Reply:
Hmm, this also was copy+pasted from DatasetType's `update` method- it seems to be working fine there? It also seems to be working fine on my branch, although I think context.getActor() is returning an incorrect URN right now.